keep/providers/kibana_provider/kibana_provider.py

```diff
@@ -434,6 +441,11 @@ def setup_webhook(
         self.logger.info("Done setting up webhooks")
 
     def validate_config(self):
+        if self.is_installed or self.is_provisioned:
+            host = self.config.authentication['kibana_host']
+            host = "https://" + host if not (host.startswith("http://") or host.startswith("https://")) else host
+            self.config.authentication['kibana_host'] = host
+
```

Comment:
I left this transformation logic as it is because its specifically for previously installed providers either through API or provisioning. If I move the logic to the `KibanaProviderAuthConfig` class, it will be implemented on new inputs as well and we do not want that.